oncall/autoIssueAssign.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+autoAssign() {
+  curl -X POST "https://api.github.com/repos/microsoft/Oryx/issues/$3/assignees" -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $2" -d "{\"assignees\": ["\"$1\""]}"
+}
+
+checkCustomOncall() {
+  input=$1
+  today=$2
+  token=$3
+  issue=$4
+  currentOncall=$5
+  if [ -z "$input" ]; then
+    return 1
+  fi
+
+  IFS="," read -ra arr <<< "$1"
```

Comment:
nit: it may be worth putting a comment above this line that highlights the format of a line in the custom on-call rotation list